SocialNetwork/SocialNetwork.Core/Models/Post.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SocialNetwork.Core.Models
+{
+    public class Post
+    {
+        public int Id { get; set; }
+        public string? Description { get; set; }
+        public object? Media { get; set; }
+        public long LikesCount { get; set; }
+        public long DislikesCount { get; set; }
```

Comment:
In the task it says to stora Likes. We don't want user to add Dislikes so we don't need to store them